app/src/providers/ContractProvider.tsx

```diff
@@ -23,17 +28,36 @@ export const ContractProvider: React.FC = ({ children }) => {
       const applicationAddr = await addressFromName('application.artistry.test');
       const registryAddr = await addressFromName('registry.artistry.test');
       const artistsAddr = await addressFromName('artists.artistry.test');
+      const arrAddr = await addressFromName('arr.artistry.test');
+      const royaltyAddr = await addressFromName('royalty.artistry.test');
+      const eursAddr = await addressFromName('eurs.artistry.test');
 
       const governance = new web3.eth.Contract(Governance.abi, governanceAddr);
       const artifactApplication = new web3.eth.Contract(ArtifactApplication.abi, applicationAddr);
       const artifactRegistry = new web3.eth.Contract(ArtifactRegistry.abi, registryAddr);
       const artists = new web3.eth.Contract(Artists.abi, artistsAddr);
+      const arr = new web3.eth.Contract(ARRRegistry.abi, arrAddr);
+      const royalty = new web3.eth.Contract(RoyaltyDistributor.abi, royaltyAddr);
+      const eurs = new web3.eth.Contract(ERC20Eurs.abi as AbiItem[], eursAddr);
+
+      console.log('Contracts provided:', {
```

Comment:
Debug console.log

Reply:
That's a good one. Need to know the loaded contract addresses

Reply:
Okay cool